code-challenges.test.js

```diff
@@ -26,42 +26,162 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // Pseudo code:
+    // Input: an object
+    // Output: a string that states whether the object is divisible by three, or is not divisible by three
+    // Process: create a function using % 3 === 0 operator to determine if a number is divisible by 3
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
+describe("threeDivider", () => {
+    it(" takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+    const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
+    const object2 = { number: 0 }
 // Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
+    const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
+        expect(threeDivider(object1.number)).toEqual("15 is divisible by three")
+        expect(threeDivider(object2.number)).toEqual("0 is divisible by three")
+        expect(threeDivider(object3.number)).toEqual("-7 is not divisible by three")
```

Comment:
Make sure you are paying attention to the prompt! The function should take an object as an argument, not a number, which is what you are passing it in these tests.